components/raftstore/src/store/fsm/peer.rs

```diff
@@ -3177,14 +3186,14 @@ where
         }
     }
 
-    fn reset_raft_tick(&mut self, state: GroupState) {
+    fn reset_raft_tick_with_missing(&mut self, state: GroupState, missing_ticks: usize) {
         debug!(
             "reset raft tick to {:?}", state;
             "region_id"=> self.fsm.region_id(),
             "peer_id" => self.fsm.peer_id(),
         );
         self.fsm.reset_hibernate_state(state);
-        self.fsm.missing_ticks = 0;
+        self.fsm.missing_ticks = missing_ticks;
```

Comment:
Seems this can violate the raft leadership assumption. In this case it is possible that the awaken leader is still in lease and then a follower starts a election(since missing ticks is raft_max_election_timeout_ticks) and this election can succeed (vote from the other follower). Thus, there will be 2 leader